scripts/install-macos.sh

```diff
@@ -0,0 +1,268 @@
+    rm -f "$TEMP_DIRECTORY/go.tar.gz"
+}
+
+# Once we have Golang; finish the install processes inside the development directory to avoid errors (Glide)
+function cloneIntoDir() {
+    echo "Setting up Kelp folders in the Golang working directory"
+    
+    # check github.com/stellar/kelp
+    if [ -d "$GOPATH/github.com/stellar/kelp" ]; then
+        echo "Kelp dir exists, no need to clone."
+    else
+        # pwd
+        mkdir -p $GOPATH/github.com/stellar/kelp
+        echo "Cloning Kelp into $GOPATH/src/github.com/stellar/kelp"
+        git clone https://github.com/stellar/kelp.git $GOPATH/src/github.com/stellar/kelp
+    fi
+
+    cd $GOPATH/src/github.com/stellar/kelp
+}
+
+# After Golang install Glide
+function installGlide() {
+    echo "Installing Glide."
+    if curl --version; then
+        curl https://glide.sh/get | sh
+    elif wget --version; then
+        wget https://glide.sh/get | sh
+    else
+        echo "curl and wget are not available, install glide manually https://github.com/Masterminds/glide"
+        exit
+    fi
+
+    glide install
+}
+
+# After Glide install Astilectron
+function installAstilectron() {
+    go get -u github.com/asticode/go-astilectron-bundler/... 
+    go install github.com/asticode/go-astilectron-bundler/astilectron-bundler
+}
+
+function postGoInstall() {
+    cloneIntoDir
+    isGlide
+    installAstilectron
+
+    checkDeps
+    echo "Remember, PostgreSQL must be running to store data."
+    echo "Remember, Docker and CCXT must be configured for the expanded set of priceFeeds and orderbooks."
+    echo "Run everything related to Kelp in BASH shell only."
+    echo "All dependencies successfully installed. Run build script from $GOPATH/src/github.com/stellar/kelp/scripts/build.sh"
+}
+
+# Utility function for checking dependency statuses
+function checkDeps(){
+    isNode
+    if [ $OK ]; then
+        echo "Node `node -v` is installed"
+    else
+        echo "Node is not installed"
+    fi
+
+    isYarn
+    if [ $OK ]; then
+        echo "Yarn `yarn -v` is installed"
+    else
+        echo "Yarn is not installed"
+    fi
+
+    isGit
+    if [ $OK ]; then
+        echo "`git version` is installed"
+    else
+        echo "Git is not installed"
+    fi
+
+    isGo
+    if $OK; then
+        echo "`go version` is installed"
+    else
+        echo "Goland is not installed"
+    fi
+}
+
+
+#**********************
+# Execute the functions
+#**********************
+cd
+
+isNode
+if [ $OK ]; then
+    echo "Node is installed on your machine."
+else
+    echo "Node is not installed on your machine!"
+    exit 1
+fi
+
+isYarn
+if [ $OK ]; then
+    echo "Yarn is installed on your machine."
+else
+    echo "Yarn is not installed on your machine!"
+    exit 1
+fi
+
+isGit
+if [ $OK ]; then
+    echo "Git is installed on your machine."
+else
+    echo "Git is not installed on your machine!"
+    exit 1
+fi
+
+isGo
+if $OK; then
+    echo "true for some reason"
```

Comment:
this echo statement seems confusing. we don't need to echo anything here since we echo the go version below.